src/__tests__/openapi.test.js

```diff
@@ -1,33 +1,26 @@
-import { readFileSync } from 'fs';
-import { Validator } from 'jsonschema';
 import request from 'supertest';
 
 import createApp from './app';
+import validator, { openapi2, openapi3 } from './validate';
```

Comment:
I added validation of OpenAPI to more of the tests and factored out the validation test code.